e2e/i18n-fixture.ts

```diff
@@ -0,0 +1,24 @@
+import { createI18nFixture } from "playwright-i18next-fixture";
+import en from "../web/src/i18n/en";
+import fi from "../web/src/i18n/fi";
```

Comment:
Would it be possible to import these as NPM dependencies instead of using an external reference? Something like this (not 100% sure about the syntax):

```json
{
  "devDependencies": {
    "translations/en": "file:../web/src/i18n/en" // <-- Not sure if extension
  }
}
```